### ISSUE TITLE
Attempted import error: 'react-native-push-notification' does not contain a default export 

### ISSUE THREAD

**Issue by johnW6** [zo0r/react-native-push-notification#1790]
# Bug

Code: 
`import PushNotification from 'react-native-push-notification';
import NotificationHandler from './NotificationHandler';

export default class NotificationService {
    constructor(onRegister, onNotification) {
        this.lastId = 0;
        this.lastChannelCounter = 0;

        this.createDefaultChannels();

        NotificationHandler.attachRegister(onRegister);
        NotificationHandler.attachNotification(onNotification);

        // Clear badge number at start
        PushNotification.getApplicationIconBadgeNumber(function (number) {
            if (number > 0) {
                PushNotification.setApplicationIconBadgeNumber(0);
            }
        });........`

error 
`Attempted import error: 'react-native-push-notification' does not contain a default export (imported as 'PushNotification').
`

**Comment by Dallas62** [comment]
Hi @johnW6 

What's your library version, I'm not able to reproduce.

Regards,

**Comment by johnW6** [comment]
HI @Dallas62 , 

I am using "^6.1.3"

**Comment by Dallas62** [comment]
Hum, this is really strange, you are the first one which is opening this kind of issue.

Did you try to reinstall dependencies ?

**Comment by johnW6** [comment]
I did now, yarn remove, yarn add and I am still getting:

`/home/jan/code/my-project/NotificationService.js
Attempted import error: 'react-native-push-notification' does not contain a default export (imported as 'PushNotification').`

**Comment by johnW6** [comment]
When I try to run it in browser i am getting:  
`×
TypeError: Cannot assign to read only property 'exports' of object '#<Object>'
Module.<anonymous>
node_modules/react-native-push-notification/index.js:609
  606 |   return this.callNative('setNotificationCategories', arguments);
  607 | }
  608 | 
> 609 | module.exports = Notifications;
  610 | `

**Comment by Dallas62** [comment]
Which platform are you using ?
This library is made for iOS and Android, not web.

**Comment by johnW6** [comment]
Yeah I am trying to run it in Android. There I have still error about import.

**Comment by Dallas62** [comment]
This seems to be an issue with your metro bundler / compiler

**Comment by drevera** [comment]
hey @jan-ambroz I had the same error, did you find solution for that?

**Comment by jan-ambroz** [comment]
> hey @jan-ambroz I had the same error, did you find solution for that?

no, i just switch to Flutter.
